tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
```

Comment:
we have this way not written to group new things with libyui and separate it from old things, so we should not use x11/ folder. We should document this when we have a clear idea if we can reuse code for modules and installation/migration in the future.
let's put for now all in `tests/installation/scc` wdyt?

Reply:
In my view this module not related with installation but function module, since we use needle and libyui check yast scc UI page, should belong x11 directory.

Reply:
we could create our own directory for x11 under yast, but there was an idea some months ago  about having each squad its own code, at the end is the most practical, except for some library functions that might be useful sometimes. Please, for the future consider always do under some yast folder so we can find the code faster and follow this approach. The compartmentalization is IMO very important for us for the maintenance of the code.